pipert2/core/base/routine.py

```diff
@@ -177,6 +177,16 @@ def start(self) -> None:
             self.runner = self.runner_creator()
             self.runner.start()
 
+    @events(UNLINK)
+    def unlink(self, unlink_routine_name):
+        """Unlink a routine from the current routine.
+
+        Args:
+            unlink_routine_name: The name of the unlink routine.
+
+        """
+        self.message_handler.output_queue.unregister(unlink_routine_name)
```

Comment:
Also, this would work **ONLY** with the `queue_message`, which makes it kind of bad (I think at least :see_no_evil:)...

Maybe think of a better way to implement this feature generically.